Dockerfile-jekyll

```diff
@@ -0,0 +1,9 @@
+FROM ruby:2.5-stretch
+USER root
+WORKDIR /root
```

Comment:
Fails on start because unpriv jekyll can't write to .jekyll-metadata which it insists on placing in the source folder and said folder will be owned at runtime by the local UID of the user who checked out from git; see https://github.com/jekyll/jekyll/issues/3595